plugin/ToolBar.qml

```diff
@@ -22,49 +22,39 @@ import Sailfish.Silica 1.0
 PanelBackground {
     id: toolbar
 
-    property var flickable: undefined
-    property int parentHeight
-    property bool hidden
+    property Item flickable
+    property bool forceHidden
     property bool autoHide: true
+    property bool trackMovement: true
```

Comment:
What about autoShowHide ? Too ugly maybe, but contains auto which suggests well the behaviour.